akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -95,15 +112,41 @@ Scala
 Java
 :  @@snip [BasicClusterExampleTest.java](/akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/BasicClusterExampleTest.java) { #cluster-leave-example }
 
-## Serialization
-
-See [serialization](https://doc.akka.io/docs/akka/current/scala/serialization.html) for how messages are sent between
-ActorSystems. Actor references are typically included in the messages,
-since there is no `sender`. To serialize actor references to/from string representation you will use the `ActorRefResolver`.
-For example here's how a serializer could look for the `Ping` and `Pong` messages above:
-
-Scala
-:  @@snip [PingSerializer.scala](/akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/PingSerializer.scala) { #serializer }
-
-Java
-:  @@snip [PingSerializerExampleTest.java](/akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/PingSerializerExampleTest.java) { #serializer }
+ 
+## Higher level Cluster tools
+ 
+@@include[cluster.md](../includes/cluster.md) { #cluster-singleton } 
+See @ref:[Cluster Singleton](cluster-singleton.md).
+ 
+@@include[cluster.md](../includes/cluster.md) { #cluster-sharding }  
+See @ref:[Cluster Sharding](cluster-sharding.md).
+ 
+@@include[cluster.md](../includes/cluster.md) { #cluster-ddata } 
+See @ref:[Distributed Data](distributed-data.md).
+
+@@include[cluster.md](../includes/cluster.md) { #cluster-routers } 
+The group routers work as is in cluster. See @ref:[Routers](routers.md).
+
+@@include[cluster.md](../includes/cluster.md) { #cluster-pubsub } 
+The API is [https://github.com/akka/akka/issues/26338](https://github.com/akka/akka/issues/26338) and
+currently only in classic @ref:[Distributed Publish Subscribe in Cluster](../distributed-pub-sub.md).
+ 
+@@include[cluster.md](../includes/cluster.md) { #cluster-multidc }  
+The API for multi-dc sharding is [https://github.com/akka/akka/issues/27705](https://github.com/akka/akka/issues/27705),
```

Comment:
The API for multi-dc sharding is **not implemented yet**
?

same for pubsub above